lib/interface/cli/commands/runtimeEnvironments/get-images.cmd.js

```diff
@@ -0,0 +1,36 @@
+const _ = require('lodash');
+const Command = require('../../Command');
+const RuntimeEnvironmentsImages = require('../../../../logic/entities/RuntimeEnvironmentsImages');
+const Output = require('../../../../output/Output');
+require('../../defaults');
+const CFError = require('cf-errors'); // eslint-disable-line
+const getRoot = require('../root/get.cmd');
+const { sdk } = require('../../../../logic');
+require('../../../../logic/entities/Environment');
+
+const command = new Command({
+    command: 'runtime-environment-images [name]',
```

Comment:
we might want to think about some better naming for this command (and also for the API endpoint) to indicate that this is not the final images of the runtime but the images from the parent/base runtime, just in case we would like to also add the functionality of getting a list of the final RE images, and also to avoid any possible confusion.